spec/std/concurrent_spec.cr

```diff
@@ -41,4 +41,8 @@ describe "concurrent" do
     spawn method_named("foo"), name: "foo"
     Fiber.yield
   end
+
+  it "accepts method call with receiver" do
+    typeof(spawn String.new)
```

Comment:
Maybe do something real in the spec?

Reply:
I don't think there's a need for an elaborate test, the basic functionality is already tested. This spec is just to ensure the macro produces valid code.